tests/concurrent-web-server.scm

```diff
@@ -0,0 +1,102 @@
+;; Fibers: cooperative, event-driven user-space threads.
+
+;;;; Copyright (C) 2016 Free Software Foundation, Inc.
```

Comment:
This patch looks newer than that, is this line up to date?

Reply:
I just copied that from another test. I've added a line with 2022 as the date now.